### PR TITLE
Add PR Workflow instructions

### PR DESCRIPTION
**Pull request by khorne3**
[Preview](https://app.gitbook.com/@shiftleft/s/shiftleft-staging/~/drafts/-M-MYnpU3BH3EE5yG-oE/v/pr-workflow/inspect/pr-workflow)

### CODE REVIEW

File: inspect/pr-workflow.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+**Note**: If you'd prefer to use Jenkins instead of CircleCI, please review the [video instructions](https://app.gitbook.com/@shiftleft/s/shiftleft-staging/~/drafts/-M-MYnpU3BH3EE5yG-oE/v/pr-workflow/inspect/pr-workflow) for doing so.
+
+## Prerequisites
+
+We assume that you have [CircleCI](https://circleci.com/docs/2.0/first-steps/) and [GitHub](https://github.com/) accounts and that you're using Git as the version control system for your source code.
+
+## Step 1: Create the GitHub Repository
+
+You will need to have a GitHub repository that contains your source code, and your repository should include the **[.circleci/config.yml](https://circleci.com/docs/2.0/configuration-reference/) file** that CircleCI looks for when building your application.
```
Comment (arunmecheri) [inline inspect/pr-workflow.md]: file is in bold.. need to move it out of the markup

File: inspect/pr-workflow.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+You will need to make the following changes to the repository where you will be pushing your code changes:
+
+1. [Enable](https://circleci.com/docs/2.0/enable-checks/) the GitHub Checks CircleCI setting
+
+2. Set your branch protection rules. To do so, navigate to your GitHub repository. Go to **Settings** > **Branches** and under the **Branch protection rules** area, click **Add rule**
+
+3. Enter `*` under Branch name pattern to make sure that the rule applies to all branches
+
+4. Under **Protect matching branches**, check the **Require status checks to pass before merging** option and select the status checks that you want to be required under the **Status checks found in the last week for this repository **heading
```
Comment (arunmecheri) [inline inspect/pr-workflow.md]: markup not working

**Comment by khorne3** [comment]
@arunmecheri Fixed both MD formatting instances

File: inspect/pr-workflow.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ You will need to make the following changes to the repository where you will be
 
 2. Set your branch protection rules. To do so, navigate to your GitHub repository. Go to **Settings** > **Branches** and under the **Branch protection rules** area, click **Add rule**
 
-3. Enter `*` under Branch name pattern to make sure that the rule applies to all branches
+3. Enter `*` under **Branch name pattern** to make sure that the rule applies to all branches
 
 4. Under **Protect matching branches**, check the **Require status checks to pass before merging** option and select the status checks that you want to be required under the **Status checks found in the last week for this repository **heading
```
Comment (arunmecheri) [inline inspect/pr-workflow.md]: MD issue

Comment (arunmecheri) [reply inspect/pr-workflow.md]: for "status checks found in the last week for this repository". I think there's a missing space in the MD?

Comment (khorne3) [reply inspect/pr-workflow.md]: @arunmecheri Oh, I see now -- the preview rendered so I didn't see it initially

**Pull request merged**